eth/tools/_utils/normalization.py

```diff
@@ -0,0 +1,506 @@
+import binascii
+from collections.abc import (
+    Iterable,
+    Mapping,
+)
+import functools
+
+from cytoolz import (
+    assoc_in,
+    compose,
+    concat,
+    curry,
+    identity,
+    merge,
+)
+import cytoolz.curried
+
+from eth_utils import (
+    apply_formatters_to_dict,
+    big_endian_to_int,
+    decode_hex,
+    is_0x_prefixed,
+    is_bytes,
+    is_hex,
+    is_integer,
+    is_string,
+    is_text,
+    to_bytes,
+    to_canonical_address,
+    to_dict,
+    ValidationError,
+)
+import eth_utils.curried
+
+from eth.constants import (
+    CREATE_CONTRACT_ADDRESS,
+)
+
+from eth.tools._utils.mappings import (
+    deep_merge,
+    is_cleanly_mergable,
+)
+
+
+#
+# Primitives
+#
+@functools.lru_cache(maxsize=1024)
+def normalize_int(value):
```

Comment:
So we now have two copies of every method in this module:
- one in `eth/tools/_utils/normalization.py`
- one in `eth/tools/fixtures/normalization.py`

Can't these be deleted out of the `fixtures/normalization` module? Otherwise I'm worried the files are going to diverge, and maintenance will be a nightmare.

Reply:
Yeah, that wasn't intentional, probably just an accidental leftover from when I shuffled that stuff around a while back.  Removing the ones from `fixtures/normalization` seems correct.

Reply:
writing up an issue.